subgraphs/friend-tech/schema.graphql

```diff
@@ -0,0 +1,806 @@
+  netVolumeUSD: BigDecimal!
+
+  " Daily number of buy trades of subject shares by the trader "
+  dailyBuyCount: Int!
+
+  " Total number of buy trades of subject shares by the trader "
+  cumulativeBuyCount: Int!
+
+  " Daily number of sell trades of subject shares by the trader "
+  dailySellCount: Int!
+
+  " Total number of sell trades of subject shares by the trader "
+  cumulativeSellCount: Int!
+
+  " Daily number of trades of subject shares by the trader "
+  dailyTradesCount: Int!
+
+  " Total number of trades of subject shares by the trader "
+  cumulativeTradesCount: Int!
+
+  " Timestamp in which this route is created or first time used "
+  createdTimestamp: BigInt!
+
+  " Block number in which this route is created or first time used "
+  createdBlockNumber: BigInt!
+}
+
+##################################
+##### Transaction-Level Data #####
+##################################
+
+interface Event {
+  " event.transaction.hash.concatI32(event.logIndex) "
+  id: Bytes!
+
+  " Transaction hash of the transaction that emitted this event "
+  hash: Bytes!
+
+  " Event log index. For transactions that don't emit event, create arbitrary index starting from 0 "
+  logIndex: Int!
+
+  " The protocol this transaction belongs to "
+  protocol: Protocol!
+
+  " Address that received the tokens "
+  to: Bytes!
+
+  " Address that sent the tokens "
+  from: Bytes!
+
+  " Block number of this event "
+  blockNumber: BigInt!
+
+  " Timestamp of this event "
+  timestamp: BigInt!
+}
+
+type Trade implements Event @entity(immutable: true) @transaction {
+  " event.transaction.hash.concatI32(event.logIndex) "
+  id: Bytes!
+
+  " Transaction hash of the transaction that emitted this event "
+  hash: Bytes!
+
+  " Event log index. For transactions that don't emit event, create arbitrary index starting from 0 "
+  logIndex: Int!
+
+  " The protocol this transaction belongs to "
+  protocol: Protocol!
+
+  " Address that received the tokens "
+  to: Bytes!
+
+  " Address that sent the tokens "
+  from: Bytes!
+
+  " Trader address"
+  trader: Trader!
+
+  " Subject address"
+  subject: Subject!
+
+  " trade type - BUY / SELL "
+  type: TradeType!
+
+  " Number of shares "
+  shares: BigInt!
+
+  " price of shares "
+  sharePriceETH: BigInt!
+  sharePriceUSD: BigDecimal!
+
+  " protocol fee "
+  protocolFeeETH: BigInt!
+  protocolFeeUSD: BigDecimal!
+
+  " subject fee "
+  subjectFeeETH: BigInt!
+  subjectFeeUSD: BigDecimal!
+
+  " total trade amount (sharePrice + protocolFee + subjectFee) "
+  amountETH: BigInt!
+  amountUSD: BigDecimal!
+
+  " Block number of this event "
+  blockNumber: BigInt!
+
+  " Timestamp of this event "
+  timestamp: BigInt!
+}
+
+##################################
+############# Helpers ############
+##################################
+
+# Helps to accumulate total unique users
+type _Account @entity @regularPolling {
+  " Address of the Account "
+  id: Bytes!
+}
+
+# Helper entity for calculating daily active users
```

Comment:
Could some of these activeX be combined into 1 entity? Like `_ActiveEntity` or will there be conflicts?